### PR TITLE
Add toggle for terminal and allow dot-sourcing for powershell.execute

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
This is a continuation of https://github.com/coc-extensions/coc-powershell/pull/73 by @mikeTWC1984

with some minor tweaks.

It adds the ability to toggle the integrated console with `powershell.toggleTerminal` and adds a new setting:

```
"powershell.integratedConsole.executeInCurrentScope"
```

which, when `true` (`false` is the default) will dot source a script in the session instead of letting it run in its own scope.

### CODE REVIEW

**Comment by TylerLeonhardt** [comment]
This also revs to version `0.1.0` to prep for a release. We can say this is little less alpha at this point :)

File: package.json [hunk 1]
```diff
@@ -5,12 +5,16 @@
...
     "readme": "README.md",
-    "version": "0.0.22",
+    "version": "0.1.0",
     "publisher": "yatli, tylerl0706, corbob",
     "repository": {
         "type": "git",
         "url": "https://github.com/yatli/coc-powershell"
     },
+    "bugs": {
+        "url": "https://github.com/mikeTWC1984/coc-powershell-patched/issues"
```
Comment (yatli) [inline package.json:15]: use our urls

File: src/client/process.ts [hunk 2]
```diff
@@ -122,16 +122,34 @@ export class PowerShellProcess {
...
 
-    public async showTerminalIfVisible() {
+    public async showTerminalIfNotVisible() {
         if (this.consoleTerminal) {
             const winid: number = await vscode.workspace.nvim.eval(`bufwinid(${this.consoleTerminal.bufnr})`) as number;
 
-            // If winid is -1, it means the window is not visible/is hidden.
-            if (winid > -1) {
-                this.consoleTerminal.show(!this.config.integratedConsole.focusConsoleOnExecute);
+            // Show terminal if it's hidden when running F5/F8 or if focusConsoleOnExecute,
```
Comment (yatli) [inline src/client/process.ts:129]: remove explicit key bindings in comments

Comment (TylerLeonhardt) [reply src/client/process.ts]: changed!

File: package.json (hunk 1 above)
Comment (TylerLeonhardt) [reply package.json]: fixed!

**Pull request merged**